chapter2.adoc

```diff
@@ -44,41 +44,27 @@ When `mdbgen` is set to 0, the <<dbops, debug operations>> are disallowed and th
 ==== Supervisor Domain Debug Control
 The Smsdedbg extension cite:[smmtt] introduces `sdedbgalw` field (bit 7) in CSR <<Sdseccsr,msdcfg>>, to control the debuggability of supervisor domains. When `mdbgen`=0, the `sdedbgalw` field determines both the debug allowed privilege modes and the <<dbgaccpriv, debug access privilege>>, as illustrated in <<dbgpriv>>. 
 
+[[dbgaccpriv]]
+==== Debug Access Privilege
+The *debug access privilege* is defined as the privilege level granted to the external debugger when performing state accesses via the hart, such as Abstract Commands and Program Buffer execution. Memory and register accesses from Debug Mode carry *debug access privilege* instead of always with M-mode. The privilege is determined by <<mdbgctl, `mdbgen`>> and <<submdbgctl, `sdedbgalw`>>, as listed in <<dbgpriv>>. The `dmprv` field in `dcsr` modifies the memory load/store privilege, while leaving the register access privilege unaffected. When `dmprv`=1, the memory load and store uses the privilege level represented in `prv` and `v` fields. Any attempt by the debugger to access state inaccessible to *debug access privilege* will return an error and set `abstractcs.cmderr` to 3.
```

Comment:
Better to create a separate section for dmprv

Reply:
The `dmprv` is detailed in below CSR section. It is mentioned here to highlight its effect to memory accesses.